File: bootstrap/src/test/java/net/dontdrinkandroot/wicket/bootstrap/TestFormPanel.java
package net.dontdrinkandroot.wicket.bootstrap;

import org.apache.wicket.markup.html.form.Form;
import org.apache.wicket.markup.html.panel.Panel;


public class TestFormPanel extends Panel
{

	private Form<Void> form;


	public TestFormPanel(String id)
	{
		super(id);
		this.form = new Form<Void>("form");
		this.add(this.form);
	}

	public Form<Void> getForm()
	{
		return this.form;
	}
}

File: bootstrap/src/test/kotlin/net/dontdrinkandroot/wicket/bootstrap/test/TestFormPanel.kt
package net.dontdrinkandroot.wicket.bootstrap.test

import org.apache.wicket.markup.html.form.Form
import org.apache.wicket.markup.html.panel.Panel

class TestFormPanel(id: String) : Panel(id) {

    val form: Form<Void> = Form("form")

    init {
        this.add(form)
    }
}